package.json

```diff
@@ -34,7 +34,8 @@
   },
   "devDependencies": {
     "prettier": "^2.5.0",
-    "typescript": "^5.1.3"
+    "tslib": "^2.6.2",
```

Comment:
`tslib` should be moved to `dependencies` https://zenzes.me/today-i-learned-tslib-is-not-a-devdependency/

Reply:
I moved it back to the shared package then. It does get hoisted to some weird version though.